beacon_localization:
    scan_topic: base_scan
    post_distance: 1.25    # Distance posts are apart from one another on beacon 
    left_post_loc: [-0.65, 0]
    right_post_loc: [0.65, 0]